File: collapsingtoolbarlayout-subtitle/tests/src/com/hendraanggrian/collapsingtoolbarlayout/subtitle/test/InstrumentedActivity.java
package com.hendraanggrian.collapsingtoolbarlayout.subtitle.test;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

/**
 * @author Hendra Anggrian (hendraanggrian@gmail.com)
 */
public class InstrumentedActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_instrumented);
    }
}

File: collapsingtoolbarlayout-subtitle/tests/src/com/hendraanggrian/collapsingtoolbarlayout/subtitle/activity/InstrumentedActivity.kt
package com.hendraanggrian.collapsingtoolbarlayout.subtitle.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class InstrumentedActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(com.hendraanggrian.collapsingtoolbarlayout.subtitle.test.R.layout.activity_instrumented)
    }
}